cishouseholds/pipeline/pipeline_stages.py

```diff
@@ -132,6 +132,7 @@ def table_to_table(
     alternate_prefix: str = None,
     alternate_database: str = None,
     transformation_functions: List[str] = [],
+    latest_table: bool = True,
```

Comment:
All good to go but think default value should be False

Reply:
right you are!